third_party/move/tools/move-package/tests/test_runner.rs

```diff
@@ -81,9 +81,9 @@ fn run_test_impl(
                 )
                 .into())
             },
-            (true, _) => match BuildPlan::create(resolved_package)
-                .and_then(|bp| bp.compile_no_exit(&compiler_config.clone(), &mut Vec::new()))
-            {
+            (true, _) => match BuildPlan::create(resolved_package).and_then(|bp| {
+                bp.compile_no_exit(&compiler_config.clone(), vec![], &mut Vec::new())
```

Comment:
The juxtaposition of two different ways to create an empty vector is kind of odd.  How about
```
.., vec![], &mut vec![])
```